parsec/backend/organization.py

```diff
@@ -124,9 +126,36 @@ async def api_organization_status(self, client_ctx, msg):
             }
         )
 
-    @api("organization_stats", handshake_types=[APIV1_HandshakeType.ADMINISTRATION])
+    @api("organization_stats", handshake_types=[HandshakeType.AUTHENTICATED])
     @catch_protocol_errors
     async def api_organization_stats(self, client_ctx, msg):
```

Comment:
naming should be `api_authenticated_organization_stats`

Reply:
done